palindrome.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+#include <math.h>
+#include <string.h>
+
+int Palindrome_test(char *inputString, int leftpart, int rightpart);
```

Comment:
Nice job, but just a few test cases you missed: 
1. Empty string - should return that it's a palindrome.
2. Capitals - Hannah returned not a palindrome when it is! 
3. sentences that are palindromes! "car rac" is a palindrome but isn't treated as such

Make sure to thoroughly test your code!